src/main/java/uk/gov/hmcts/reform/pip/data/management/service/ChannelManagementService.java

```diff
@@ -38,4 +37,20 @@ public String requestFileGeneration(UUID artefactId) {
             return "";
         }
     }
+
+    public void deleteFiles(UUID artefactId) {
+        try {
+            webClient.delete()
+                .uri(url + "/publication/" + artefactId)
+                .attributes(clientRegistrationId("channelManagementApi"))
+                .retrieve()
+                .bodyToMono(String.class)
+                .block();
+        } catch (WebClientException ex) {
+            log.error(writeLog(
```

Comment:
Can we log the artefact ID here in the error? I think if it fails it'd be good to make sure we can trace it back to the specific artefact

Reply:
Now logged the artefact ID